src/site/content/en/blog/compute-pressure/index.md

```diff
@@ -0,0 +1,284 @@
+observer.unobserve();
+```
+
+### Exposing limited information via thresholds and buckets
+
+One of the key API design goals is to provide enough information for an
+application to make useful decisions based on compute pressure, while not
+exposing information that might
+[harm a user's privacy](https://github.com/oyiptong/compute-pressure/blob/main/security-privacy-self-assessment.md).
+
+With this in mind, the API reports back normalized, unitless values between
+`0.0` and `1.0` instead of more identifiable values like clock speeds in GHz.
+The values passed to the callback fall into one of the "buckets" defined by the
+thresholds used to configure the observer. The actual value will be the midpoint
+between the upper and lower bounds for the bucket—in other words, they will be
+values determined by how you configure the observer, and not based on the actual
+device.
+
+To give an example, the configuration in the preceding section uses
+`cpuUtilizationThresholds: [0.5, 0.75, 0.9]`. This configuration sets up a total
+of four possible buckets that the reported values might fall into, covering the
+ranges 0.0 to 0.5, 0.5 to 0.75, 0.75 to 0.9, and 0.9 to 1.0. To represent bucket
+membership, the value that is reported to the callback is the average between
+the lower and upper bounds for the bucket.
+
+<figure class="w-figure">
+  {% Img src="image/FNkVSAX8UDTTQWQkKftSgGe9clO2/VUrr3MykViLDtHuTkjYG.svg", alt="A diagram illustrating the bucketing for different actual CPU utilizations", width="800", height="549" %}
+  <figcaption class="w-figcaption">
+    This diagram illustrates the relationship between three thresholds and the
+    four buckets that they form.
+  </figcaption>
+</figure>
+
+Here's how some "real" CPU utilization values, as reported by the operating
+system to the browser, would be represented as values passed to your callback:
+
+- "Real" 0% utilization would map to the 0.0 to 0.5 bucket, and would be
+  reported as `cpuUtilization: 0.25`, since 0.25 is the average of 0.0 and 0.5.
+- "Real" 40% utilization would similarly map to the 0.0 to 0.5 bucket, and be
+  reported as `cpuUtilization: 0.25`
+- "Real" 60% utilization would map to the 0.5 to 0.75 bucket, and be reported as
+  `cpuUtilization: 0.625`
+- "Real" 80% utilization would map to the 0.75 to 0.9 bucket, and be reported as
+  `cpuUtilization: 0.825`
+
+The same threshold and bucket system applies to the exposed CPU speed values.
+
+{% Aside %}
+The values reported by the API reflect the overall state of the CPU,
+due to utilization across all applications currently running. Therefore, even if
+your own web application is almost entirely idle, you may see high utilization
+and clock speed values reported, due to the activities of other applications.
+{% endAside %}
+
+## Try a demo
+
+When accessed on a browser that supports the origin trial (currently Chrome 92
+or later), this [demo page](https://oyiptong.github.io/compute-pressure-demo/)
```

Comment:
This URL will likely be updated once the OT starts and we can move to a new repo.